src/Native/System.Native/pal_process.cpp

```diff
@@ -6,6 +6,11 @@
 
 #include <assert.h>
 #include <errno.h>
+#if HAVE_LINUX_PATH_MAX
+#include <linux/limits.h>
+#else
+#include <iostream>
```

Comment:
Hmm, plain limits.h works on Linux too (at least on my machine) and it is supposed to be there according to http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/limits.h.html